quickwit/quickwit-telemetry/src/payload.rs

```diff
@@ -76,6 +76,10 @@ pub enum TelemetryEvent {
     RunService(String),
     /// EndCommand (with the return code)
     EndCommand { return_code: i32 },
+    /// Uptime timer has elapsed.
+    UptimeCheckIn,
+    /// UI home page was loaded.
+    UiVisited,
```

Comment:
I would go for `UiRequest`.

@guilload I know you like renaming, don't hesitate to suggest some names :)